$OpenBSD: patch-configure_ac,v 1.1 2015/01/09 17:34:52 sthen Exp $

autoconf's check for these succeeds, but they still don't exist (and the
second one fails at build time).

--- configure.ac.orig	Fri Jan  9 17:22:21 2015
+++ configure.ac	Fri Jan  9 17:26:44 2015
@@ -64,8 +64,6 @@ DX_PS_FEATURE(OFF)
 DX_INIT_DOXYGEN([lldpd], [doxygen.cfg], [doxygen])
 
 # Check some compiler flags
-AX_CFLAGS_GCC_OPTION([-fdiagnostics-show-option])
-AX_CFLAGS_GCC_OPTION([-fdiagnostics-color=auto])
 AX_CFLAGS_GCC_OPTION([-pipe])
 AX_CFLAGS_GCC_OPTION([-Wall])
 AX_CFLAGS_GCC_OPTION([-W])
